gradio/blocks.py

```diff
@@ -506,11 +517,14 @@ def __init__(
         if theme is None:
             theme = DefaultTheme()
         elif isinstance(theme, str):
-            try:
-                theme = Theme.from_hub(theme)
-            except Exception as e:
-                warnings.warn(f"Cannot load {theme}. Caught Exception: {str(e)}")
-                theme = DefaultTheme()
+            if theme.lower() in BUILT_IN_THEMES:
```

Comment:
Not related to your pr but let's add theme to the docstring (didn't realize it was missing) and explain it can be a string (built-in theme or upstream theme) or theme instance

Reply:
Nit, but being able to pass a str for a built in theme should go on the changelog

Reply:
Sounds good! Will make these changes